app/src/main/java/com/eveningoutpost/dexdrip/GcmListenerSvc.java

```diff
@@ -542,8 +544,12 @@ public void run() {
                 } else if (action.equals("ssom")) {
                     if (Home.get_master()) {
                         if (payload.equals("challenge string")) {
-                            UserError.Log.e(TAG, "Stopping sensor by remote");
-                            StopSensor.stop();
+                            if (Pref.getBoolean("plus_accept_follower_actions", true)) {
+                                UserError.Log.e(TAG, "Stopping sensor by remote");
```

Comment:
Logging sensor stop by remote as an error is also not correct because the stop is intentional. I would categorize it as info or at maximum as a warning.

Reply:
Is info "i"?

Reply:
Wait a minute!
That's not my code.  That was there already.  You are asking me to change someone else's code.  You know that, right?

Reply:
Done

Reply:
> That's not my code. That was there already. You are asking me to change someone else's code. You know that, right?

Indeed, yes. The whole file is not your code. That's open source. :smirk: